build.rs

```diff
@@ -21,6 +22,10 @@ fn main() {
         out_path,
         "dynamodb",
     );
+    serde_generate(
```

Comment:
This is so nit-picky but while you're in here can you put the KMS gen and serde_gen together, and the Dynamo bits together?  Right now they are interleaved.